TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -283,5 +281,15 @@ private DatetimeConstant CreateDatetimeConstant(CodeElementsParser.Datetime_cons
 
             return null;
         }
+        private SqlColumnName CreateSqlColumnName(CodeElementsParser.Column_nameContext context)
+        {
+            if (context.UserDefinedWord() != null)
```

Comment:
Why adding the test now ? In previous code there was no nullity check. To improve the code, you should add tests for the from clause with missing column names and see how ANTLR handles it. Then you will find how to adapt the builder. The goal is to have a syntax error from ANTLR and no exception in the builder.

For example, in ExecSqlWithFromClause.rdz.cbl, try this statement:

```
SELECT * FROM t_table as new_name (c1,,)
```

The syntax here is wrong, so we should one or more errors from ANTLR and the builder should create a list with one new column name (for 'c1').

Reply:
The test can now be safely removed as we check the absence of diagnostics before calling the method.